Rater.API/Controllers/SpaceController.cs

```diff
@@ -28,12 +30,11 @@ public async Task<ActionResult<SpaceResponseDto>> AddSpace(GrandSpaceRequestDto
 
         }
 
-
-        [HttpGet("GetAllSpaces")]
+        [HttpPost("GetSpaceByLink"), Authorize(Policy = "SpaceIdentify")]
```

Comment:
Do we really need this `Authorize(Policy = "SpaceIdentify")`

Reply:
I think we need it because we might not want to allow access to someone who is not authenticated to the space.